app/src/components/market/market_view_container.tsx

```diff
@@ -88,7 +91,7 @@ const MarketViewContainer: React.FC<Props> = (props: Props) => {
       setHash(hash)
     }
 
-    get24hsVolume()
+    provider && get24hsVolume()
```

Comment:
PP: I don't like this idiom (`condition && statement-with-side-effects`). Besides... `provider` is not nullable here, is it? If you got an error with it being undefined, then we have a typing problem somewhere.